soh/src/code/code_800F9280.c

```diff
@@ -695,39 +695,42 @@ u8 func_800FAD34(void) {
     return D_80133418;
 }
 
-void func_800FADF8(void) {
-    u8 playerIdx, j;
-
-    for (playerIdx = 0; playerIdx < 4; playerIdx++) {
-        D_8016E348[playerIdx] = 0;
-        D_8016E750[playerIdx].unk_254 = NA_BGM_DISABLED;
-        D_8016E750[playerIdx].unk_256 = NA_BGM_DISABLED;
-        D_8016E750[playerIdx].unk_28 = 0;
-        D_8016E750[playerIdx].unk_18 = 0;
-        D_8016E750[playerIdx].unk_14 = 0;
-        D_8016E750[playerIdx].unk_258 = 0;
-        D_8016E750[playerIdx].unk_4D = 0;
-        D_8016E750[playerIdx].unk_4E = 0;
-        D_8016E750[playerIdx].unk_250 = 0;
-        D_8016E750[playerIdx].unk_252 = 0;
-        for (j = 0; j < 4; j++) {
-            D_8016E750[playerIdx].volScales[j] = 0x7F;
+void Audio_ResetActiveSequences(void) {
+    u8 seqPlayerIndex;
+    u8 scaleIndex;
+
+    for (seqPlayerIndex = 0; seqPlayerIndex < 4; seqPlayerIndex++) {
+        sNumSeqRequests[seqPlayerIndex] = 0;
+
+        gActiveSeqs[seqPlayerIndex].seqId = NA_BGM_DISABLED;
+        gActiveSeqs[seqPlayerIndex].prevSeqId = NA_BGM_DISABLED;
+        gActiveSeqs[seqPlayerIndex].tempoTimer = 0;
+        gActiveSeqs[seqPlayerIndex].tempoOriginal = 0;
+        gActiveSeqs[seqPlayerIndex].tempoCmd = 0;
+        gActiveSeqs[seqPlayerIndex].channelPortMask = 0;
+        gActiveSeqs[seqPlayerIndex].setupCmdNum = 0;
+        gActiveSeqs[seqPlayerIndex].setupFadeTimer = 0;
+        gActiveSeqs[seqPlayerIndex].freqScaleChannelFlags = 0;
+        gActiveSeqs[seqPlayerIndex].volChannelFlags = 0;
+        for (scaleIndex = 0; scaleIndex < 4; scaleIndex++) {
+            gActiveSeqs[seqPlayerIndex].volScales[scaleIndex] = 0x7F;
         }
-        D_8016E750[playerIdx].volFadeTimer = 1;
-        D_8016E750[playerIdx].fadeVolUpdate = 1;
+
+        gActiveSeqs[seqPlayerIndex].volFadeTimer = 1;
+        gActiveSeqs[seqPlayerIndex].fadeVolUpdate = true;
     }
 }
```

Comment:
all the other changes are just what was needed to be able to pull this in and have everything still build/work